src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/RefAPISource.java

```diff
@@ -277,6 +293,8 @@ private Genomics createGenomicsService(final PipelineOptions pipelineOptions) {
         }
     }
 
+    // TODO: Move these to a CustomCoder. That will allow us to do something else (possibly better) for Spark.
+    // TODO: See Issue #849.
```

Comment:
Yea, it's a little more work that I want to bite off. I do expect to get to this later (plus we have the issue to track it).